src/SmartComponents/Inventory/InventoryDetail.scss

```diff
@@ -34,8 +34,7 @@
   .ins-entity-facts {
     div {
       span:first-child {
-        font-weight: var(--pf-global--FontWeight--bold);
-        margin-right: 15px;
+        margin-right: 0.4rem;
```

Comment:
Can we use the `rem` mixin here? Just to stay consistent.

Reply:
Sure thing!